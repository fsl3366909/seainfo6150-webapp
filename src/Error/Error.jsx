import React from 'react'
import err from './404.jpg'
import './index.css'
const Error = () => {
    return (
        <div>
            <h1>error</h1>
            <img src={err} alt="" />
        </div>
    )
}

export default Error
