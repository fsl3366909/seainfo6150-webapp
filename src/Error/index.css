img {
  width: 400px;
  display: block;
  margin: auto;
}
