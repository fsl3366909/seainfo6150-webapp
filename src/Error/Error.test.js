import React from "react";
import Error from "./Error.jsx";

it("renders the home component correctly", () => {
    const { container } = render(<Error />);
    expect(container).toMatchSnapshot();
});
