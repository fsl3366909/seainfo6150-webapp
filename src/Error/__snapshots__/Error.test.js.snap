// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home component correctly 1`] = `
<div>
  <div>
    <h1>
      error
    </h1>
    <img
      alt=""
      src="404.jpg"
    />
  </div>
</div>
`;
